proxy.go

```diff
@@ -472,9 +504,20 @@ func (p *Proxy) doPost(ctx context.Context, wg *sync.WaitGroup, destination stri
 	)...)
 }
 
+func (p *Proxy) ReportRuntimeMetrics() {
+	mem := &runtime.MemStats{}
+	runtime.ReadMemStats(mem)
+	metrics.ReportBatch(p.TraceClient, []*ssf.SSFSample{
+		ssf.Gauge("mem.heap_alloc_bytes", float32(mem.HeapAlloc), nil),
```

Comment:
i think i might also like to see the number of live objects in a gauge, here - `mem.Mallocs - mem.Frees`

Reply:
That should probably be a histogram? Maybe?

Regardless, I'm going to leave this out (since it's a nit) mostly because veneur doesn't emit this metric and I'm kinda lazy. :)

Reply:
no worries. if we do get to it later, we could choose to roll it up as a gauge directly (which would be the appropriate type if we do the subtraction op ourselves, i think), or we could emit both the mallocs and frees as distinct count metrics, and do the op later. The latter option certainly gives more flexibility, so is probably better.